doc/uftrace-live.md

```diff
@@ -322,6 +322,46 @@ Each field has following meaning:
 The default value is 'duration,tid'.  If given field name starts with "+", then it'll be appended to the default fields.  So "-f +time" is as same as "-f duration,tid,time".  And it also accepts a special field name of 'none' which disables the field display and shows function output only.
 
 
+DYNAMIC TRACING
+===============
+The uftrace tool supports dynamic function tracing which can be enabled and disabled at runtime on x86_64.  Before recording functions, you can build the target program with `-pg`, then it might have a tiny performance impact because all funtions call `mcount()`.  So you can partially enable given functions at load time with the `-P`/`--patch` option, after building the target program with not only `-pg` but also `-mfentry -mnop-mcount` to add 5 byte nop at function prologue.
+
+The following example shows a error message when normally running uftrace with the excutable built with `-pg -mfentry -mnop-mcount`. Because 'mcount' symbol can't be found.
```

Comment:
The last sentence can be changed to:

Because the binary doesn't call any instrumentation code (i.e. 'mcount').